test/unittests/test_dd_synthesis.cpp

```diff
@@ -0,0 +1,72 @@
+#include "algorithms/synthesis/dd_synthesis.hpp"
+#include "algorithms/synthesis/encoding.hpp"
+#include "core/io/pla_parser.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace dd::literals;
+using namespace syrec;
+
+class TestDDSynth: public testing::TestWithParam<std::string> {
+protected:
+    TruthTable                     tt{};
+    std::string                    testCircuitsDir = "./circuits/";
+    std::unique_ptr<dd::Package<>> ddSynth         = std::make_unique<dd::Package<>>(9U);
```

Comment:
```suggestion
    auto ddSynth         = std::make_unique<dd::Package<>>(9U);
```

Reply:
I'm getting the following error "auto' not allowed in non-static class member".

Reply:
Right. Forgot you can't do that. Feel free to resolve this.